dense_map/geometry_mapper/readme.md

```diff
@@ -630,14 +630,19 @@ reference documentation in
 
 and also in build_map.md in that repository. 
 
+If the map to be built is large, consider using the Theia SfM
+software. Its usage is described in:
+
+    https://nasa.github.io/astrobee/html/theia_map.html
```

Comment:
Suggestion: we can use href for this:
[Theia map](https://nasa.github.io/astrobee/html/theia_map.html)

Reply:
It actually parsed my comment, click edit on the comment to see the raw :)